shell/components/SortableTable/index.vue

```diff
@@ -343,14 +348,18 @@ export default {
     }
 
     return {
-      currentPhase:               ASYNC_BUTTON_STATES.WAITING,
+      refreshButtonPhase:         ASYNC_BUTTON_STATES.WAITING,
       expanded:                   {},
       searchQuery,
       eventualSearchQuery,
       subMatches:                 null,
       actionOfInterest:           null,
       loadingDelay:               false,
       debouncedPaginationChanged: null,
+      /**
+       * The is the bool the DOM uses to show loading state. it's proxied from `loading` to avoid blipping the indicator (see usages)
+       */
+      pLoading:                   false,
```

Comment:
We were going to rename this to `isLoading`

Reply:
Good point! Updated